tests/fixtures.py

```diff
@@ -10,20 +10,26 @@
 from __future__ import unicode_literals
 
 import pytest
+import six
 
 from dockerfile_parse import DockerfileParser
 
 
-@pytest.fixture(params=[False, True])
+@pytest.fixture(params=[(dockerfile, cache_content) for dockerfile in ['fileobj', 'path'] for cache_content in [True, False]])
 def dfparser(tmpdir, request):
     """
 
     :param tmpdir: already existing fixture defined in pytest
     :param request: parameter, cache_content arg to DockerfileParser
     :return: DockerfileParser instance
     """
-    tmpdir_path = str(tmpdir.realpath())
-    return DockerfileParser(tmpdir_path, request.param)
+
+    if request.param[1] == 'path':
```

Comment:
We could unpack the tuple as `use_fileobj, cache_content = request.param` to make it a little more readable.